src/gmprocess/subcommands/projects.py

```diff
@@ -140,8 +143,7 @@ def main(self, gmrecords):
             if not prompt.query_yes_no(question, default="yes"):
                 sys.exit(0)
 
-            shutil.rmtree(conf_path, ignore_errors=True)
```

Comment:
@emthompson-usgs are things at data_path no longer being removed here?

Reply:
I suppose not. Perhaps we need to fix that, but I think it is a separate issue. I'm note sure the "delete" argument ever gets used by anyone.

Reply:
@gferragu I modified the shutil to delete the parent of config_path so that it would also delete the project folder. See[ ](https://github.com/usgs/groundmotion-processing/pull/1013/files#:~:text=.parent%2C-,ignore_errors,-%3DTrue)) I don't know if this is the right approach (probably not) so if the alternative way of deleting the config_path and data_path folders separately is preferred I can change this.

Reply:
There's no guarantee that the the data and conf directories share a common parent directory (although that is the default), which is why it done separately before.

Reply:
@emthompson-usgs OK I didn't think about this because I am using the default. I can revert it back, but I think I'll wait until the Windows relative_path issue is resolved.